packages/plugins/users-permissions/admin/src/hooks/useFetchRole/index.js

```diff
@@ -1,13 +1,14 @@
 import { useCallback, useReducer, useEffect, useRef } from 'react';
 import { useNotification } from '@strapi/helper-plugin';
+import { getFetchClient } from '@strapi/admin/admin/src/utils/getFetchClient';
 import reducer, { initialState } from './reducer';
-import axiosInstance from '../../utils/axiosInstance';
 import pluginId from '../../pluginId';
 
 const useFetchRole = (id) => {
   const [state, dispatch] = useReducer(reducer, initialState);
   const toggleNotification = useNotification();
   const isMounted = useRef(null);
+  const { get } = getFetchClient();
```

Comment:
Why are you using `getFetchClient` over `useFetchClient`?

Reply:
because I can't use an hook inside a function

Reply:
But you can compose hooks.

Reply:
yes sorry, my bad, I thought was related to an util and not to another hook, I have resolved the error